test/rss.cpp

```diff
@@ -323,6 +323,191 @@ TEST_CASE("rss_item::sort_flags() cleans up flags", "[rss]")
 	}
 }
 
+TEST_CASE("rss_feed::sort() correctly sorts articles", "[rss]")
+{
+	configcontainer cfg;
+	cache rsscache(":memory:", &cfg);
+	rss_feed f(&rsscache);
+	for (int i = 0; i < 5; ++i) {
+		const auto item = std::make_shared<rss_item>(&rsscache);
+		item->set_guid(std::to_string(i));
+		f.add_item(item);
+	}
+
+	SECTION("title")
+	{
+		auto articles = f.items();
+		articles[0]->set_title("Article you must read");
+		articles[1]->set_title("Another great article");
+		articles[2]->set_title("A boring article");
+		articles[3]->set_title("Read me");
+		articles[4]->set_title("Wow tests are great");
```

Comment:
I know, right? :rofl:

Seriously, though: that's already sorted by title, so it doesn't really test that title-asc is working. Can you reorder a few items, please?